include/gz/sim/components/Model.hh

```diff
@@ -75,10 +75,18 @@ namespace serializers
         }
       }
 
+      if (!skip)
+      {
       _out << "<?xml version=\"1.0\" ?>"
            << "<sdf version='" << SDF_PROTOCOL_VERSION << "'>"
            << (skip ? std::string() : modelElem->ToString(""))
            << "</sdf>";
```

Comment:
add indentation, and since `skip` will always be false here, we can simplify the logic?

```suggestion
        _out << "<?xml version=\"1.0\" ?>"
             << "<sdf version='" << SDF_PROTOCOL_VERSION << "'>"
             << modelElem->ToString("")
             << "</sdf>";
```